cmd/fetchbranches/SemaphoreCI.go

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) 2017 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Semaphore environment variables https://semaphoreci.com/docs/available-environment-variables.html
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+)
+
+type SemaphoreCI struct{}
+
+// GetPR retrieves the PR number from the Semaphore environment
+func (ci *SemaphoreCI) GetPR() (int, error) {
```

Comment:
How about introducing a PR object instead of returning an int? That object could have an `id` and a `URL` for instance with methods on it instead of global methods on the github oject: github.GetPRAuthorComments() becomes PR.GetAuthorComments()